minidump-stackwalk/stackwalker.cc

```diff
@@ -127,6 +128,16 @@ static string ToHex(uint64_t value) {
   return buffer;
 }
 
+static string ToHexAddress(uint64_t value, int arch_bits) {
```

Comment:
I did it this way because maybe some day there will be 128 bit systems or something. Now that I say that, it seems silly to think about that especially given we're probably switching to the new minidump-stackwalker soon.

I didn't spend time on truncating values. This only pads out values. I couldn't find a crash report that needed truncation to test with which I think means there aren't many. I also figured we're only sticking with this minidump-stackwalk for a little while longer so it's kind of moot.